utils/mentions.py

```diff
@@ -23,22 +23,26 @@ def __init__(self, type, id, string):
     def is_id_type(self):
         return self.type == MentionType.ID
 
-    def type_str(self):
-        if self.type == MentionType.ID:
-            return "id"
 
-        return "string"
+class MakeMention:
+    def id_mention(id):
+        return Mention(MentionType.ID, id, None)
+
+    def string_mention(string):
+        return Mention(MentionType.STRING, None, string)
```

Comment:
These methods can live on the `MentionType`, as [`@staticmethod`s](https://docs.python.org/3/library/functions.html#staticmethod).

Reply:
It's worth noting that the first parameter of a method in Python is the class it is defined on. In this case, if you had an instance of a `MakeMention` then `id` and `string` would both be that instance if invoked on the instance. the `staticmethod` decorator avoids this.